Snowplow/Internal/Configurations/SPTrackerConfiguration.m

```diff
@@ -48,6 +48,8 @@ - (instancetype)initWithDictionary:(NSDictionary<NSString *,NSObject *> *)dictio
         if (devicePlatform) {
             self.devicePlatform = SPStringToDevicePlatform(devicePlatform);
         }
+        // TODO: Uniform "base64encoding" string on both Android and iOS trackers
+        self.base64Encoding = [dictionary sp_boolForKey:@"base64encoding" defaultValue:self.base64Encoding];
```

Comment:
Does this need adding to https://github.com/snowplow/iglu-central/pull/1118 before release?

Reply:
At the moment the schema is a subset of the configuration actually possible in the tracker.
I don't know if we need to add it now.
Initially I thought to increase the configurable fields with minor versions of the schema, it shouldn't cause compatibility issues.
Btw, I'm open to change of strategy.

Reply:
yeah thats a good point. I think seeing this configuration value being fixed after pointed it out to me but it's fine to extend it further in the future. Good to perhaps keep track of missing configuration in an issue to see if missing configuration options become an issue for anyone.